core-feature-pack/src/main/resources/modules/system/layers/base/org/jboss/as/cli/main/module.xml

```diff
@@ -35,6 +35,25 @@
     </resources>
 
     <dependencies>
+        <!-- JAXP default dependencies. DO NOT REMOVE!!!
+                   These are loaded by the JAXP redirect facility in jboss-modules when this module is used
+                   as a boot module (this is the case for server processes in domain mode). The purpose is
```

Comment:
I picked this change from Tomaz Cerar repository.
It's copy paste from org.jboss.as.server.main module AFAICS.
The purpose of this commit is CLI should always use OUR specified JAXP providers
and don't rely on those provided by JDK. I can update the comment accordingly.